packages/react-resizable-panels/package.json

```diff
@@ -57,7 +58,7 @@
     "@vitest/ui": "^3.1.2",
     "eslint": "^8.37.0",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "jsdom": "^24.0.0",
+    "happy-dom": "^17.4.6",
```

Comment:
it probably would work with jsdom but I switched at some point while trying out different things and I have not bothered to switch this back